.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ on:
     branches: [ main ]
 
 env:
-  GO_VERSION: 1.17.7
+  GO_VERSION: 1.18
```

Comment:
should we test with both 1.17 and 1.18?

Reply:
The more coverage the better, but it should be in a different PR.

Reply:
i will work on that